client/my-sites/stats/stats-list/stats-list-card.jsx

```diff
@@ -86,13 +105,12 @@ const StatsListCard = ( {
 							data={ item }
 							maxValue={ barMaxValue }
 							hasIndicator={ item?.className?.includes( 'published' ) }
-							onClick={ ( e ) => localClickHandler( e, item ) }
-							leftSideItem={
-								item?.countryCode && <StatsListCountryFlag countryCode={ item.countryCode } />
-							}
-							rightSideItem={ rightSideItem }
+							onClick={ localClickHandler }
+							leftSideItem={ leftSideItem }
+							rightSideItem={ ( incomingItem ) => getRightItem( incomingItem, index ) }
```

Comment:
What do we gain by passing this template function through? Would it make more sense to just pass the `rightSideItem` through directly instead of a template function like this?

Reply:
I was passing an item directly but it turned out that if I pass the function I can render the component recursively (for groups) and pass current child data as a parameter to generate actions for the given child, not the top-level parent.

Reply:
Ah, good point!

Could we possibly rename this property so that it's more obvious that `rightSideItem` should be a function and not a node?